crates/bevy_math/src/common_traits.rs

```diff
@@ -0,0 +1,133 @@
+use glam::{Quat, Vec2, Vec3, Vec3A, Vec4};
+use std::fmt::Debug;
+use std::ops::{Add, Div, Mul, Sub};
+
+/// A type that supports the mathematical operations of a vector space, irrespective of dimension.
```

Comment:
I would prefer if the docs for this trait were more like `Eq`: We should clearly state that this implies mathematical properties that cannot be checked by the compiler.

I'm sure there some of our less mathematical users would also appreciate it if we restated the basic vector space properties here.